internal/cmd/on-call.go

```diff
@@ -45,7 +46,7 @@ var onCallCmd = &cobra.Command{
 
 		user, err := client.GetCurrentUser(pagerduty.GetCurrentUserOptions{})
 		if err != nil {
-			return err
+			return wrap.Error(err, "It seems like the authtoken is not set correctly or outdated. Please update the authtoken in the .pd.yml file. If you don't know how to create your authtoken, this might help:\n https://support.pagerduty.com/docs/generating-api-keys#generating-a-personal-rest-api-key\n")
```

Comment:
```suggestion
			return wrap.Error(err, "it seems like the authtoken is not set correctly or outdated. Please update the authtoken in the .pd.yml file. If you don't know how to create your authtoken, this might help:\n https://support.pagerduty.com/docs/generating-api-keys#generating-a-personal-rest-api-key\n")
```